RubeesEat/Model/InMemoryBillRepository.cs

```diff
@@ -29,4 +29,32 @@ public void Add(Bill bill)
         ArgumentNullException.ThrowIfNull(bill);
         _bills.Add(bill.Id, bill);
     }
+
+    public IReadOnlyList<Bill> GetAllForUser(Guid userGuid)
+    {
+        return (from bill in _bills
+            where Enumerable.Any(bill.Value.EntryLines, line => line.Person.Id == userGuid)
+            select bill.Value).ToList();
+    }
+
+    public IReadOnlyList<BalanceChange> GetRecentBalanceChanges(Guid userGuid, int amount)
+    {
+        var billsWithUser = GetAllForUser(userGuid).ToList();
+        billsWithUser = billsWithUser.OrderByDescending(bill => bill.Date).ToList();
```

Comment:
Once you've look at LINQ and `IEnumerable<T>` a bit can you tell me what seems wrong to me here?

Reply:
Is the takeaway, that we didn't need to make it into a List because IEnumerable already has OrderByDescending?